### PR TITLE
feat(docs): document Token.nullValue()

### PR DESCRIPTION
**Pull request by ansgarm**

### CODE REVIEW

File: website/docs/cdktf/concepts/tokens.mdx [hunk 1]
```diff
@@ -59,3 +60,33 @@ Later in synthesis, CDKTF will resolve the token to `${module.<module id>.public
 ```
 
 Refer to the [AWS CDK documentation](https://docs.aws.amazon.com/cdk/latest/guide/tokens.html) for more detailed information about tokens.
+
+### Using Terraform's `null` value
+Some edge cases require passing the Terraform `null` value to for example an attribute of a resource.
```
Comment (ansgarm) [inline website/docs/cdktf/concepts/tokens.mdx:65]: @laurapacilio, what do you think?

File: website/docs/cdktf/concepts/tokens.mdx [hunk 2]
```diff
@@ -59,3 +60,33 @@ Later in synthesis, CDKTF will resolve the token to `${module.<module id>.public
...
+
+<CodeTabs>
```
Comment (DanielMSchmidt) [inline website/docs/cdktf/concepts/tokens.mdx:67]: We should add these to the documentation examples, but we can do it in a follow up

**Review by DanielMSchmidt — APPROVED** [review]

File: website/docs/cdktf/concepts/tokens.mdx (hunk 2 above)
Comment (ansgarm) [reply website/docs/cdktf/concepts/tokens.mdx:67]: Yeah, I agree – I copied them from the edge provider tests for now 😁

Comment (ansgarm) [reply website/docs/cdktf/concepts/tokens.mdx:68]: Follow-up: https://github.com/hashicorp/terraform-cdk/issues/2298
Might be easiest to do this when we have documentation examples for all languages in place

**Pull request merged**